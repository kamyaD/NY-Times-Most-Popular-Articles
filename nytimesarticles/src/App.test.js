
import ReactDOM from 'react-dom';
import App from './App';

import renderer from 'react-test-renderer'; 

it('renders without crashinng', ()=>{
  const div=document.createElement('div');
  ReactDOM.render(<App/>, div);
  ReactDOM.unmountComponentAtNode(div);
})

it("matches snapshot", ()=>{
  const tree = renderer.create(<App/>).toJSON(); 
  expect(tree).toMatchSnapshot();
})