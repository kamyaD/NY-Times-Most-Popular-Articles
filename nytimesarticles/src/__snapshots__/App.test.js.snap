// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="card"
  >
    <h2>
      NY Times Most Read
    </h2>
  </div>
</div>
`;
